[2024-03-17T10:49:58.057+0500] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_rocket_launches.notify manual__2024-03-17T05:29:14.689333+00:00 [queued]>
[2024-03-17T10:49:58.065+0500] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_rocket_launches.notify manual__2024-03-17T05:29:14.689333+00:00 [queued]>
[2024-03-17T10:49:58.065+0500] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-03-17T10:49:58.190+0500] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): notify> on 2024-03-17 05:29:14.689333+00:00
[2024-03-17T10:49:58.193+0500] {standard_task_runner.py:60} INFO - Started process 23649 to run task
[2024-03-17T10:49:58.196+0500] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'download_rocket_launches', 'notify', 'manual__2024-03-17T05:29:14.689333+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/download_rocket_launches.py', '--cfg-path', '/tmp/tmpemen0z4h']
[2024-03-17T10:49:58.197+0500] {standard_task_runner.py:88} INFO - Job 22: Subtask notify
[2024-03-17T10:49:58.345+0500] {task_command.py:423} INFO - Running <TaskInstance: download_rocket_launches.notify manual__2024-03-17T05:29:14.689333+00:00 [running]> on host muhammadsami-Latitude-E6540
[2024-03-17T10:49:58.900+0500] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='download_rocket_launches' AIRFLOW_CTX_TASK_ID='notify' AIRFLOW_CTX_EXECUTION_DATE='2024-03-17T05:29:14.689333+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-17T05:29:14.689333+00:00'
[2024-03-17T10:49:58.904+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-17T10:49:58.906+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "There are now $(ls GettingStarted/data/images/ | wc -l) images."']
[2024-03-17T10:49:58.915+0500] {subprocess.py:86} INFO - Output:
[2024-03-17T10:49:58.918+0500] {subprocess.py:93} INFO - ls: cannot access 'GettingStarted/data/images/': No such file or directory
[2024-03-17T10:49:58.919+0500] {subprocess.py:93} INFO - There are now 0 images.
[2024-03-17T10:49:58.920+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-17T10:49:59.035+0500] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=download_rocket_launches, task_id=notify, execution_date=20240317T052914, start_date=20240317T054958, end_date=20240317T054959
[2024-03-17T10:49:59.211+0500] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-17T10:49:59.227+0500] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
